gravitee-am-common/src/main/java/io/gravitee/am/common/web/UriBuilder.java

```diff
@@ -140,19 +143,18 @@ public static UriBuilder fromHttpUrl(String httpUrl) {
      * Convert a String to the application/x-www-form-urlencoded MIME format
      */
     public static String encodeURIComponent(String s) {
-        String result;
-        try {
-            result = URLEncoder.encode(s, "UTF-8");
-        } catch (UnsupportedEncodingException e) {
-            result = s;
+        if (s == null) {
+            return null;
         }
+        String result;
+        result = URLEncoder.encode(s, StandardCharsets.UTF_8);
```

Comment:
UTF-8 is guaranteed to be supported on every JVM implementation at least since Java 7:
<img width="917" alt="image" src="https://github.com/user-attachments/assets/b9ba7c8d-5236-4757-8af0-4b70bfbe4b73">

Also constants in StandardCharsets are Charsets, not Strings, so the UnsupportedCharsetException cannot be thrown anymore. I'll fix the other place to match this one.